platform/viewer/cypress/integration/common/OHIFUserPreferences.spec.js

```diff
@@ -0,0 +1,571 @@
+describe('OHIF User Preferences', () => {
+  context('Study List Page', function() {
+    before(() => {
+      cy.visit('/');
+    });
+
+    beforeEach(() => {
+      // Open User Preferences modal
+      cy.openPreferences().then(() => {
+        cy.initPreferencesModalAliases();
+      });
+    });
+
+    it('opens User Preferences modal and verify the displayed information', function() {
+      cy.get('@preferencesModal').should('be.visible');
+      cy.get('.HotKeysPreferences').should('be.visible');
+      cy.get('@hotkeysLink').should('have.class', 'active');
+      cy.get('@generalLink').should('be.visible');
+      cy.get('@restoreBtn')
+        .scrollIntoView()
+        .should('be.visible');
+      cy.get('@cancelBtn').should('be.visible');
+      cy.get('@saveBtn').should('be.visible');
```

Comment:
For the buttons, maybe instead of `should be visible`, we can make sure they contain the text we expect?

What is `@generalLink`? What is `@hotkeysLink`?
What is `.HotkeytsPreferences`?

The text says "opens user preferences modal", but I think that happens in the `beforeEach`
We could close it after initing aliases, or update the text here?